tests/test_hecuba.py

```diff
@@ -0,0 +1,193 @@
+import gc
+import unittest
+
+import numpy as np
+from hecuba import config
+from pycompss.api.api import compss_wait_on
+from sklearn.datasets import make_blobs
+
+import dislib as ds
+from dislib.cluster import KMeans
+from dislib.decomposition import PCA
+from dislib.neighbors import NearestNeighbors
+from dislib.regression import LinearRegression
+
+
+class HecubaTest(unittest.TestCase):
+
+    def test_iterate_rows(self):
+        """
+        Tests iterating through the rows of the Hecuba array
+        """
+        config.session.execute("TRUNCATE TABLE hecuba.istorage")
+        config.session.execute("DROP KEYSPACE IF EXISTS hecuba_dislib")
+        block_size = (20, 10)
+        x = np.array([[i] * 10 for i in range(100)])
+
+        data = ds.array(x=x, block_size=block_size, backend="hecuba",
+                        name="hecuba_dislib.test_array")
+
+        for i, chunk in enumerate(data._iterator(axis="rows")):
+            r_data = chunk.collect()
+            r_x = np.array([[j] * 10
+                            for j in range(i * block_size[0],
+                                           i * block_size[0] + block_size[0])])
+            self.assertTrue(np.array_equal(r_data, r_x))
+
+        self.assertEqual(i + 1, len(data._blocks))
+
+    def test_fit_predict(self):
```

Comment:
Each test should have a short description at the top.

Reply:
Descriptions added